evento/src/api.js

```diff
@@ -214,5 +214,29 @@ export default {
 		catch(err) {
 			return { success: false, error: SOMETHING_WENT_WRONG_ERROR };
 		}
+	},
+
+	async getAuthenticationStatus() {
+		if(!session.isLoggedIn()) {
+			return { success: true, payload: { isAuthenticated: false} };
+		}
+
+		try {
+			const response = await fetch('/authentication');
+			if(!response.ok) {
+				return await _createErrorResult({ from: response, defaultValues: { type: "auth", message: INVALID_CREDENTIALS_MESSAGE } });
+			}
+
+			const json = await response.json();
+			const authenticated = json.authenticated;
+			if(!authenticated) {
+				session.reset(); // reset cookies
```

Comment:
If the backend says that user is not logged in, then make sure that the cookies on the client side reflects that